styles/globals.css

```diff
@@ -5,7 +5,13 @@ body {
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
   background-color: rgba(207, 201, 201, 0.356);
-  height: 100%;
+  min-height: 100%;
```

Comment:
![Screenshot (474)](https://user-images.githubusercontent.com/70262783/109605319-fef1fc00-7b4a-11eb-80fb-47f05329feb5.png)

@SBagaria2710 Background color is not fixed yet.